### PR TITLE
SW-975 Add description to facility data

### PR DESCRIPTION
**Pull request by sgrimm**
The new design for seed bank management allows the user to enter both a name and a
description for each seed bank. Add a field for the description, and remove a few
unused columns from the facilities table.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #342** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/342" target="_blank">Graphite</a> 
    * **PR #340** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/340" target="_blank">Graphite</a>  👈
      * **PR #341** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/341" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/340?utm_source=stack-comment).

File: src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt [hunk 1]
```diff
@@ -488,6 +488,7 @@ class AdminController(
 
   @PostMapping("/updateFacility")
   fun updateFacility(
+      @RequestParam("description") description: String?,
```
Comment (karthikbtf) [inline src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt:491]: Do we need a similar parameter in the createFacility api? Assuming that's the API the UI would call.

**Comment by karthikbtf** [comment]
looks good, one question about creating facilities with a description.

**Comment by sgrimm** [comment]
Good point; there isn't a create endpoint at all right now! I'll add that.

File: src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt (hunk 1 above)
Comment (karthikbtf) [reply src/main/kotlin/com/terraformation/backend/customer/api/AdminController.kt:491]: having issues with github pulling in stale comments

**Comment by sgrimm** [comment]
`AdminController` is the logic for the admin UI (the barebones one you get if you go to `/admin/`). I figure it's fine to make people edit the description as a separate step in the admin UI.

**Comment by karthikbtf** [comment]
> `AdminController` is the logic for the admin UI (the barebones one you get if you go to `/admin/`). I figure it's fine to make people edit the description as a separate step in the admin UI.

Thanks!

**Comment by sgrimm** [comment]
**Graphite Merge Job**

Current status: ✅ Merged

This pull request was successfully merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/340?utm_source=stack-comment).


Job Reference: N8YQwlkvTw2MBjRc508J

**Pull request merged**